usdt/Cargo.toml

```diff
@@ -9,11 +9,11 @@ description = "Dust your Rust with USDT probes"
 repository = "https://github.com/oxidecomputer/usdt.git"
 
 [dependencies]
```

Comment:
Worth throwing a comment on here noting that the version specs are load-bearing for publishing purposes?

Reply:
Will do, good idea.

Reply:
Done in 1d10020